google-cloud-spanner/lib/google/cloud/spanner/results.rb

```diff
@@ -143,28 +157,25 @@ def rows
                 # Flush the buffered responses now that they are all handled
                 buffered_responses = []
               end
-            rescue GRPC::Cancelled, GRPC::DeadlineExceeded, GRPC::Internal,
-                   GRPC::ResourceExhausted, GRPC::Unauthenticated,
-                   GRPC::Unavailable, GRPC::Core::CallError => err
-              if resume_token.nil? || resume_token.empty?
-                # Re-raise if the resume_token is not a valid value.
-                # This can happen if the buffer was flushed.
-                raise Google::Cloud::Error.from_error(err)
-              end
+            rescue GRPC::Core::CallError,
```

Comment:
Note that `GRPC::Core::CallError` is not a `GRPC::BadStatus` error, so it won't be wrapped.